module/updatable_configs/require_approvals_for_seal_construction.go

```diff
@@ -0,0 +1,63 @@
+package updatable_configs
+
+import (
+	"sync"
+
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/module/defaults"
+	"github.com/onflow/flow-go/module/updatable_configs/validation"
+)
+
+// min number of approvals required for constructing a candidate seal
+type RequiredApprovalsForSealConstructionInstance struct {
+	sync.RWMutex
+	requiredApprovalsForSealConstruction uint
```

Comment:
Agree with [Jordan's comment](https://github.com/onflow/flow-go/pull/2518#discussion_r889329071). An atomic will have much lower runtime penalties. We can verify the new value without lock.